enterprise/frontend/src/embedding-sdk/components/public/InteractiveQuestion/components/Filter.tsx

```diff
@@ -24,10 +24,10 @@ const FilterInner = ({
 }: {
   question: Question;
 } & FilterProps) => {
-  const { onQuestionChange } = useInteractiveQuestionContext();
+  const { updateQuestion } = useInteractiveQuestionContext();
```

Comment:
`updateQuestion` updates the question instead of running the query, so it's renamed to be a bit more explicit